src/mscorlib/src/System/Enum.cs

```diff
@@ -844,20 +900,23 @@ public int CompareTo(Object target)
 
         #region Public Methods
         public String ToString(String format) {
+            char formatCh;
             if (format == null || format.Length == 0)
-                format = "G";
+                formatCh = 'G';
+            else
```

Comment:
@benaadams, with these latest changes, has there been any impact on performance since you edited the description?